zyzmoz/global-player
# ESlint, Prettier, Airbnb Config

#### **Trello ticket URL:**
[Ticket](https://trello.com/c/b8ih6NEi/18-configure-coding-practices)


#### **Description:**
_Please remember that the reviewer may not be as familiar with work being done or the project as you._

The initial configuration for the backend was created under the `backend/` folder


**Run command:**

```
cd backend \
npm install \
npm run dev
```

- [x] This code includes new npm packages/libraries

#### **QA instructions (if applicable):** 
1. Go to the backend files
2. Add some code 
3. Try to format the code

---

_Thank you for your contribution to the Global Player monorepo._
_Before submitting this PR, please make sure:_

- [x] Code cleanup (removed dead code, console logs, sorted props, etc)
- [x] Updated Trello ticket with info and linked to this PR
